src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ClicktrackingArticleEditModeTests.java

```diff
@@ -23,6 +27,10 @@
   })
   public void ClickTrackingArticleEditMode_001_verifyPreviewModalEvents() {
     WikiBasePageObject base = new WikiBasePageObject(driver);
+    base.openMainPage(wikiURL);
+    NavigationBar navbar = new NavigationBar(driver);
+    navbar.executeScript(script);
```

Comment:
it doesn't make a difference since navbar extends base
It indicates that now I deal with navigation bar
base was used only to open mainPage